src/pyseekdb/client/client_base.py

```
@@ -761,7 +781,29 @@ def get_collection(self, name: str, embedding_function: EmbeddingFunctionParam =
             collection = self._get_collection_v2(name, embedding_function)
         return collection
 
+    def _resolve_collection_metadata_from_sdk_collections(self, collection_name: str) -> _CollectionMeta | None:
+        """
+        Resolve collection metadata infromation from sdk_collections table
+        """
+        try:
+            query_sql = f"SELECT COLLECTION_ID, COLLECTION_NAME, SETTINGS FROM `{CollectionNames.sdk_collections_table_name()}` WHERE COLLECTION_NAME = '{collection_name}'"
```

Ruff (0.14.14)

Possible SQL injection vector through string-based query construction

(S608)